packages/sdk/react-client/src/testing/ClientSpaceDecorator.tsx

```diff
@@ -5,18 +5,63 @@
 import { faker } from '@faker-js/faker';
 import { DecoratorFunction } from '@storybook/csf';
 import { ReactRenderer } from '@storybook/react';
-import React, { useMemo, useState } from 'react';
+import React, { PropsWithChildren, useState } from 'react';
 
 import { Trigger } from '@dxos/async';
-import { Client, EchoSchema, Invitation, PublicKey } from '@dxos/client';
+import { EchoSchema, Invitation, Space } from '@dxos/client';
 import { TestBuilder } from '@dxos/client/testing';
 import { raise } from '@dxos/debug';
 import { log } from '@dxos/log';
-import { useAsyncEffect } from '@dxos/react-async';
 import { Loading } from '@dxos/react-components';
 
 import { ClientProvider } from '../client';
 
+const testBuilder = new TestBuilder();
+const services = () => testBuilder.createClientServicesHost();
+
+// TODO(wittjosiah): Generates warning `No peers to notarize with` during invitation, but retry succeeds.
```

Comment:
@dmaretskyi @richburdon if you have any thoughts on why this is let me know

Reply:
Yeah, warning is a non issue, we could hide it